osrparse/replay.py

```diff
@@ -271,7 +273,7 @@ def pack(self):
         data += self.pack_int(r.mods.value)
         data += self.pack_life_bar()
         data += self.pack_timestamp()
-        data += self.pack_replay_data()
+        data += events
         data += self.pack_long(r.replay_id)
 
```

Comment:
not sure why this was changed, but should be reverted

Reply:
`_Packer` needs to pack replay data first before anything else, so that the hash itself could be generated
to not call the function twice, it's saved into a variable, and then used for packing a hash & replay data